WindowManager.cs

```diff
@@ -81,11 +84,13 @@ public WindowManager(Game game, GraphicsDeviceManager graphics) : base(game)
         /// </summary>
         public int RenderResolutionY { get; private set; } = 600;
 
+#if WINFORMS
         /// <summary>
         /// Gets a boolean that determines whether the game window currently has input focus.
         /// </summary>
         public bool HasFocus { get; private set; } = true;
 
+#endif
```

Comment:
I don't think it's a good idea to remove this from non-WinForms builds, it adds a lot of extra checks to the rest of the code. Rather introduce a cross-platform version of `WindowsGameWindowManager.HasFocus()` (can just return `game.IsActive`).